lib/Buzz/Browser.php

```diff
@@ -289,7 +202,7 @@ public function getLastResponse()
         return $this->lastResponse;
     }
 
-    public function setClient(ClientInterface $client)
+    public function setClient($client)
```

Comment:
should it actually be mutable ?

Reply:
No. It should be immutable when we release this. See https://github.com/kriswallsmith/Buzz/issues/246